wgpu-hal/src/gles/mod.rs

```diff
@@ -217,7 +237,14 @@ enum TextureInner {
 impl TextureInner {
     fn as_native(&self) -> (glow::Texture, BindTarget) {
         match *self {
-            Self::Renderbuffer { raw, .. } => panic!("Unexpected renderbuffer {}", raw),
+            #[cfg(not(target_arch = "wasm32"))] // Wasm renderbuffers don't impl Debug
```

Comment:
let's not split this just because of `Debug`. It's a panic anyway, no need to increase code complexity for it